app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextActions.java

```diff
@@ -305,4 +313,183 @@ private void runPrefixReplaceAction(final Pattern actionPattern, final String ac
 
         runRegexReplaceAction(patterns);
     }
+
+
+    private static class ListLine {
```

Comment:
All of this should probably move to a separate file. But for now, its here

Reply:
I'm fine with having it in one of the existing markdown classes. However, I feel the better fit is `MarkdownAutoFormat.java`, which has exactly the use of "do something when user enters text" .... auto-heading etc.

---> ListLine,  OrderedListLine, UnOrderedListLine